tests/integration/targets/podman_prune/tasks/main.yml

```diff
@@ -0,0 +1,100 @@
+- name: Create random names
+  ansible.builtin.set_fact:
+    cname: "{{ 'ansible-container-%0x' % ((2**32) | random) }}"
+    nname: "{{ 'ansible-network-%0x' % ((2**32) | random) }}"
+    vname: "{{ 'ansible-volume-%0x' % ((2**32) | random) }}"
+
+- name: Test podman_prune module
+  block:
+    # Create objects to be pruned
+    - name: Create container
+      containers.podman.podman_container:
+        name: "{{ cname }}"
+        image: quay.io/podman/hello:latest
+        state: present
+      register: container
+    - name: Create network
+      containers.podman.podman_network:
+        name: "{{ nname }}"
+        state: present
+      register: network
+    - name: Create volume
+      containers.podman.podman_volume:
+        name: "{{ vname }}"
+        state: present
+      register: volume
+
+    # Prune objects
+    - name: Prune objects
+      containers.podman.podman_prune:
+        container: true
+        network: true
+        volume: true
+      register: result
+
+    # Analyze result
+    - name: Show results
+      ansible.builtin.debug:
+        var: result
+
+    - name: Verify assertions for network, container and volume
```

Comment:
I think would be useful to check that container, volume and network don't exist after pruning. Because in CI nobody is looking on debug print :) We can check this either by shell "podman exists volume/network/container or by info modules: `containers.podman.podman_container_info`, `containers.podman.podman_network_info`, `containers.podman.podman_volume_info` and see they have empty results.

Reply:
ack!